.circleci/config.yml.j2

```diff
@@ -740,6 +754,9 @@ commands:
       edge_build:
         description: "Indicate if this is an edge build"
         type: boolean
+      nightly_build:
+        description: "Indicate if this is a nightly build"
+        type: boolean
```

Comment:
We could probably add the default of `true` to be on the safer side, so we don't have to edit/update the non-nightlies

Reply:
If I understand this correctly, I think setting the default to `true` would force us to update the non-nightlies. Setting it to `false` would let us omit that in the non-nightlies.